app/views/layouts/application.html.erb

```diff
@@ -49,8 +49,6 @@
       <body
         data-user-status="<%= user_logged_in_status %>"
         class="<%= Settings::UserExperience.default_font.tr("_", "-") %>-article-body default-header"
-        data-pusher-key="<%= ApplicationConfig["PUSHER_KEY"] %>"
-        data-app-domain="<%= ChatChannel.urlsafe_encoded_app_domain %>"
```

Comment:
Does anyone know what this was needed for? Is it still required even with Connect gone?

Reply:
I can't find any reference to this app domain attribute anywhere, and the encoded app domain is only used here and in pusher_channels.